services/http/api/tsconfig.json

```diff
@@ -13,10 +13,11 @@
         "skipLibCheck": true
     },
     "exclude": [
-        "./tests/mocks/auth_server",
```

Comment:
It shouldn't be skipping tests. It just skips building mock_auth_server into the `./dist` folder since it has a tsconfig of it's own. In the test, it gets started with `ts-node` anyways, so I removed the redundant building.

Reply:
If you look into the built `./dist` folder, you should see the `api.spec.js` file in there, which is the test that gets run (14 tests).

Reply:
Gotcha. Whatever you changed seemed to fix it so I'm satisfied.